### PR TITLE
Add Data Umbrella: January JupyterLab event

### PR DESCRIPTION
**Pull request by mmcelaney**

### CODE REVIEW

**Comment by mmcelaney** [comment]
cc: @stefanv @reshamas

**Comment by reshamas** [comment]
Thank you @mmcelaney.
Can we keep the previous events? Referencing below discussion:
https://github.com/scientific-python/scientific-python.org/pull/706#issue-2717631872

**Comment by mmcelaney** [comment]
> Thank you @mmcelaney. Can we keep the previous events? Referencing below discussion: [#706 (comment)](https://github.com/scientific-python/scientific-python.org/pull/706#issue-2717631872)

Just added the intro to git and expressions (Polars & Narwhals) back in...

File: calendars/data-umbrella.yaml [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+    end: 2025-01-07 18:00:00 +00:00
+    url: https://www.bigmarker.com/neo4j/Data-Umbrella-Webinar
+
   - summary: Introduction to Git
     description: |
       Join us for an introductory session on Git. Discover some
       fundamental concepts, explore its practical applications, and
       learn how to efficiently use both the command line and the Visual
       Studio Code graphical user interface (GUI).
-
```
Comment (stefanv) [inline calendars/data-umbrella.yaml:9]: Quick check: were these separator lines meant to be removed? There's one below as well.

Comment (mmcelaney) [reply calendars/data-umbrella.yaml:9]: Woops - Fixed that

Comment (stefanv) [reply calendars/data-umbrella.yaml:9]: Empty lines needed, I think!

Comment (mmcelaney) [reply calendars/data-umbrella.yaml:9]: Oh - thought you were saying we needed a `-` to make the line break. I think I’ve fixed it?

**Comment by stefanv** [comment]
Thanks!

**Pull request merged**